controller/controlplane/manager_options.go

```diff
@@ -509,6 +509,27 @@ func WithDataPlaneSyncOptions(syncOptions operatorv2alpha1.ControlPlaneDataPlane
 func WithEmitKubernetesEvents(emit bool) managercfg.Opt {
 	return func(c *managercfg.Config) {
 		c.EmitKubernetesEvents = emit
+
+	}
+}
+
+// TODO: We need to support multiple label selectors for secrets/config maps in ingress controller managers
+// when we support specifying both operator level and per ControlPlane selectors:
+// https://github.com/Kong/kong-operator/issues/1863
```

Comment:
Does that issue track making sure that the Secrets/ConfigMaps watched by CPs have both the labels set by the operator ( in CLI flags / envs ) and by the CP? 

If so I think we should explicitly state that in the issue (perhaps by a dedicated acceptance criteria) or even better, create a separate issue for operator wide label selectors for these and adjust the comment. WDYT?